src/auth/auth.proto

```diff
@@ -152,10 +156,11 @@ enum Permission {
   CLUSTER_AUTH_REVOKE_USER_TOKENS                  = 142;
   CLUSTER_AUTH_ROTATE_ROOT_TOKEN                   = 147;
 
-  CLUSTER_ENTERPRISE_ACTIVATE            = 114;
-  CLUSTER_ENTERPRISE_HEARTBEAT           = 115;
-  CLUSTER_ENTERPRISE_GET_CODE            = 116;
-  CLUSTER_ENTERPRISE_DEACTIVATE          = 117;
+  CLUSTER_ENTERPRISE_ACTIVATE                        = 114;
+  CLUSTER_ENTERPRISE_HEARTBEAT                       = 115;
+  CLUSTER_ENTERPRISE_GET_CODE                        = 116;
+  CLUSTER_ENTERPRISE_DEACTIVATE                      = 117;
+  CLUSTER_ENTERPRISE_REVOKE_USER_TOKENS_ALL_CLUSTERS = 148;
```

Comment:
I usually structure these permissions as `SCOPE_SERVICE_PERMISSION` so I wonder if these ones (that are evaluated at the enterprise level) should look like `ENTERPRISE_ LICENSE_REVOKE_TOKENS`?